liquibase-core/src/main/java/liquibase/command/core/LockServiceCommandStep.java

```diff
@@ -0,0 +1,61 @@
+package liquibase.command.core;
+
+import liquibase.Liquibase;
+import liquibase.Scope;
+import liquibase.command.*;
+import liquibase.database.Database;
+import liquibase.exception.LockException;
+import liquibase.lockservice.LockService;
+import liquibase.lockservice.LockServiceFactory;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Internal command step to be used on CommandStep pipeline to create lock services.
+ */
+public class LockServiceCommandStep extends AbstractCommandStep implements CleanUpCommandStep {
+
+    protected static final String[] COMMAND_NAME = {"lockServiceCommandStep"};
+
+    private LockService lockService;
+
+    @Override
+    public List<Class<?>> requiredDependencies() {
+        return Collections.singletonList(Database.class);
+    }
+
+    @Override
+    public List<Class<?>> providedDependencies() {
+        return Collections.singletonList(LockService.class);
+    }
```

Comment:
Smells like DI in here 😎

Reply:
That's the plan! Maybe in the future we can have an @Autowired annotation 😃